guizero/PushButton.py

```diff
@@ -24,6 +24,10 @@ def __init__(self, master, command, args=None, text="Button", icon=None, pady=10
         self._current_font = "Arial"
         self._font_size = 11
         self._value = 0
+
+        if command is None:
```

Comment:
I changed the implementation to:

```python
    def change_command(self, newcommand, args=None):
        if newcommand is None:
            newcommand = lambda: None
        else:
            if args is None:
                self._command = newcommand
            else:
                # If the args LIST was not blank, allow args
                self._command = utils.with_args(newcommand, *args)
```

So that it is possible to call PushButton.change_command(None).